src/libstore/ssh.hh

```diff
@@ -14,6 +14,9 @@ private:
     const std::string host;
     bool fakeSSH;
     const std::string keyFile;
+    /**
+     * Raw bytes, not Base64 encoding.
+     */
     const std::string sshPublicHostKey;
```

Comment:
If it's raw bytes, then should it really still be `std::string` and not something like `std::vector<u8>` or something?

Reply:
Probably, but doing that better would require more changes (because we use `std::string` in a few places for this) so I think it's best we keep it future work.

Reply:
I always felt that c++ string is just the same as a std::vector<u8> in other languages.